api/e2/e2.proto

```diff
@@ -0,0 +1,783 @@
+// Copyright 2020-present Open Networking Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file defines O-RAN E2 interface, which is the interface between an eNB and RIC
+
+syntax = "proto3";
+
+package ran.e2;
+
+service E2 {
+    rpc CellConfigRequest (E2CellConfigRequestAttribute) returns (E2CellConfigReportAttribute) {}
```

Comment:
Most of this seems to be repeated from
https://github.com/OpenNetworkingFoundation/xranc/blob/master/protos/e2-interface.proto

Could we reuse rather than duplicate?
You could put this Service in the other e2 proto file, or you could reuse the types from the other file here. In either case you'd just need to import the other file in to compile-proto.sh, like gogo proto is imported

Also for this service, it might need to have streaming requests and streaming responses to spare opening and closing the connection every time - depends on the amount of traffic really